build.sbt

```diff
@@ -384,14 +384,7 @@ def flinkLibScalaDeps(scalaVersion: String, configurations: Option[String] = Non
 
 lazy val commonDockerSettings = {
   Seq(
-    dockerBaseImage       := forScalaVersion(
-      scalaVersion.value,
-      "eclipse-temurin:17-jre-jammy",
-      (
-        2,
-        12
-      ) -> "eclipse-temurin:11-jre-jammy" // jre11, cause for jdk17 minimum scala version is 2.12.15, we use 2.12.10
-    ),
+    dockerBaseImage       := "eclipse-temurin:11-jre-jammy",    // designer should run on java11 since it may run Flink in-memory-cluster, which does not support newer java and we want to have same jdk in both designer and lite-runner
```

Comment:
... to make analysis of problems with jre compatibility easier using testing mechanism and embedded server
+ Add a TODO that we want to support jre >= 17 but before that we have to remove scala 2.12 compatibility and we should handle opening of modules for spel reflectional access to class memebers